gst/nnstreamer/nnstreamer_subplugin.c

```diff
@@ -103,6 +108,7 @@ get_subplugin (subpluginType type, const char *name)
         g_critical ("Cannot dlopen %s with error %s.", fullpath, dlerror ());
         continue;
       }
+      g_ptr_array_add (handles, (gpointer) handle);
```

Comment:
Discussion (later work) : dlopen vs g_module_open
Currently nnstreamer does not have dependency of gmodule.
Is it necessary to use gmodule and remove dl-open/close?

Reply:
If it gives us some advantage, yes. (e.g., using gmodule instead of dlfcn makes the whole nnstreamer compatible with other OS?)

Reply:
I found gmodule internal code with windows, android, darwin,... So I think it is better to use it.

Reply:
If dlfcn.h is not portable, then, yes, that may be added to the future-feature-waiting-list.